lab-allie/readme.md

```diff
@@ -0,0 +1,13 @@
+# COWSAY project
```

Comment:
Good README, but make sure to be more specific in future docs, i.e. tell a user how to install ``httpie`` on their computer and use the ``http`` commands in their terminal.